core/chains/evm/config/v2/config.go

```diff
@@ -16,35 +16,36 @@ import (
 
 type Chain struct {
 	BalanceMonitorEnabled *bool
-	BlockBackfillDepth    *uint32
-	BlockBackfillSkip     *bool
+	//TODO BalanceMonitorBlockDelay
```

Comment:
@samsondav We need to include `BalanceMonitorBlockDelay`. With two `BalanceMonitor` fields, should we make it a table?
```toml
[BalanceMonitor]
Enabled = true
BlockDelay = 1
```
Also expressible as:
```toml
BalanceMonitor.Enabled = true
BalanceMonitor.BlockDelay = 1
```

Reply:
Done, but still taking feedback :rofl: 
This could be a simple/clean case to consider whether the defaults behave how we expect.